libs/gltfio/src/FFilamentAsset.h

```diff
@@ -268,9 +270,11 @@ struct FFilamentAsset : public FilamentAsset {
     std::vector<TextureSlot> mTextureSlots;
     std::vector<const char*> mResourceUris;
     NodeMap mNodeMap; // unused for instanced assets
+    ReverseNodeMap mReverseNodeMap; // unused for instanced assets
     std::vector<std::pair<const cgltf_primitive*, filament::VertexBuffer*> > mPrimitives;
     MatInstanceCache mMatInstanceCache;
     MeshCache mMeshCache;
+    tsl::robin_map<const void *, std::string> mExtras;
```

Comment:
Instead of a entity=>node map a node=>string map, can't we just have a entity=>string map?

Reply:
Not really, extras are retrieved before entities are created.

Reply:
> @romainguy @prideout any feedback to address?

It would be good to fix the "two maps" thing somehow.  Note that the cgltf node pointers are transient and become invalid when `releaseSourceData` is called, which clears out `mNodeMap` but not your new reverse node map.

Maybe we need two maps (or two passes) during the loading process, but surely it's possible to have only one map as a member variable.

Reply:
I'll double-check how it handles the described case, but maybe I'll have to add a third "squashed" map and clean-up others after loading is complete

Reply:
Okay, got your point, a very valid one.

Reply:
There are still two maps, but one is used as cache and later wiped out after loading is complete.